deltametrics/section.py

```diff
@@ -223,9 +228,14 @@ def connect(self, CubeInstance):
                                 _gottype=type(CubeInstance)))
         self.cube = CubeInstance
         self._variables = self.cube.variables
+        self._name = self._name or name or self.section_type
```

Comment:
Yeah, that seems like it would make sense though, like the `_name` is meant to mean that it is protected attribute. I could make a property `@name` with this same logic. At least then I could issue a warning in your mentioned scenario...

Let me look at it.

Reply:
Cool, I gave `_name` a setter, so that a warning is issued if you try to give a name to an already named section.